public/api/conf/1.0/schemas/state-pension.json

```diff
@@ -0,0 +1,180 @@
+          },
+          
+          {
+            "enum": ["MarriedWomensReducedRateElection"],
+            "description": "The customer has Reduced Rate Election in force at the beginning of the tax year 35 years before the tax year of State Pension Age. Current marital status required, unable to determine State Pension amounts."
+          },
+          
+          {
+            "enum": ["AmountDissonance"],
+            "description": "Contention between amounts on internal systems. There is contention between the pension amounts in the internal systems, therefore a reliable forecast cannot be provided."
+          },
+          
+          {
+            "enum": ["PostStatePensionAge"],
+            "description": "The current date is equal to or greater than the day before the customer&#8217;s State Pension date. The customer is eligible for State Pension and therefore we do not provide a forecast."
+          }
+        ],
+        "description": "A list of exclusion reasons",
+        "example": "Abroad"
+        }
+    },
+    "earningsIncludedUpTo": {
+      "type": "string",
+      "description": "The date which the calculations are based on. This is always the end date of the last posted tax year",
+      "example": "2015-04-05"
+    },
+    "amounts": {
+      "type": "object",
+      "properties": {
+        "protectedPayment": {
+          "type": "boolean",
+          "description": "Whether the customer has an amount higher than the full rate at the end of the 2015/16 tax year",
+          "example": "false"
+        },
+        "current": {
+          "type": "object",
+          "properties": {
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per week",
+              "example": "133.41"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per month",
+              "example": "580.1"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per year",
+              "example": "6961.14"
+            }
+          }
+        },
+        "forecast": {
+          "type": "object",
+          "properties": {
+            "yearsToWork": {
+              "type": "number",
+              "description": "The number of future qualifying years needed to achieve the forecast amount",
+              "example": "1"
+            },
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per week",
+              "example": "146.76"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per month",
+              "example": "638.14.1"
```

Comment:
I think you are right... I'd automated it from the XML docs and missed it when I went through the schema!